frontend/src/assets/images.ts

```diff
@@ -0,0 +1,19 @@
+import ImageType from "../types/ImageTypes";
+
+export const JUMP_MATH_LOGO: ImageType = {
+  src:
+    "https://storage.googleapis.com/jump-math-98edf.appspot.com/jump-math-logo.png",
+  alt: "jump-math-logo",
+};
+
+export const ADMIN_SIGNUP_IMAGE: ImageType = {
+  src:
+    "https://storage.googleapis.com/jump-math-98edf.appspot.com/admin-signup.png",
+  alt: "admin-signup",
+};
+
+export const TEACHER_SIGNUP_IMAGE: ImageType = {
```

Comment:
This image is kind of low-quality, I think because it's an export from Figma. When I track down the original, I'll update in Firebase!